tests/e2e/asyncOps.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {test} from '@playwright/test'
+import { FormElements } from '../page-objects/formElements'
```

Comment:
If we are using navigation for more than one page object we should look at that as an opportunity to abstract out that part of the code.